pkg/monitor/monitor/template.go

```diff
@@ -42,21 +39,21 @@ const (
 )
 
 var (
-	truePattern      config.Regexp
-	allMatchPattern  config.Regexp
-	portPattern      config.Regexp
-	tikvPattern      config.Regexp
-	pdPattern        config.Regexp
-	tidbPattern      config.Regexp
-	addressPattern   config.Regexp
-	tiflashPattern   config.Regexp
-	pumpPattern      config.Regexp
-	drainerPattern   config.Regexp
-	cdcPattern       config.Regexp
-	importerPattern  config.Regexp
-	lightningPattern config.Regexp
-	dmWorkerPattern  config.Regexp
-	dmMasterPattern  config.Regexp
+	truePattern      string
```

Comment:
```suggestion
	truePattern = "true"
```
we can initialize these string variables directly instead of using init() funciton.